### ISSUE TITLE
Workout App Missing in Paired Apple Watch

### ISSUE THREAD

**Issue by app4g** [glzawalski/WorkoutKit-Builder#1]
Hi - Just testing out and looking at the workout kit. (Don't have an actual Apple Watch to test on)
I followed your instructions. Made a new iP11 + AW Ultra pairing.. then fired up the app of yours, created the workout and then pressed present workout. (also manually opened the AW ultra simulator like your instructions said)

seems to have transferred fine (I think) but when I go to the Apple Watch simulator, I don't see a workout app. In fact, I see very few things in the list.

Am I supposed to be able to open the Apple Watch workout app and then select this workout I created using your app?

tx

**Comment by glzawalski** [comment]
Hi. Thanks for testing out my sample project 😁

WatchOS simulator does not come with the full suite of apps pre-installed on a real device. Similar to iOS, where you don't have an app for the App Store, Clock, Camera or even a Calculator. So I imagine the Workout app will continue to be missing unless Apple decides to pre-load it into the simulator due to demand (i.e. more developers are interested in WorkoutKit and it proves to be a hinderance to development).

For the time being, when you tap to try and present a built workout and the dark preview with `"Add to Apple Watch"` appears you know both the device simulators are paired up (this preview fails otherwise) and the custom workout should be transferable to a real device. That's the most I could do with the current constraints.

**Comment by app4g** [comment]
Thx for the explanation. I really didn't know what to expect.  I even fired up some other watch OS simulators to check and yeah, they are really sparse. Didn't know that.

Since I don't have an Apple Watch , this means I can't do anything more then. :-( which is pretty sad.

one more other thing, initially I tot It wasn't working cos I wasn't on the latest Xcode beta, so once I downloaded the latest beta. (15A5209g), things just didn't work anymore. Can't compile. Telling me about "BuildStep" not found in scope and actually, it just couldn't find a whole load of stuffs. Even after I cleaned the build and re-build it still don't work.

I'm still planing to download the beta (15A5160n) which did work and try it out again (it's a big download) to see if there are changes in Xcode which caused this error.

Tx again for sharing this.
